docs/source/development/set_up_vscode.md

```diff
@@ -10,6 +10,10 @@ At this stage, you should be able to see the `conda` environment that you have c
 
 ![](../meta/images/vscode_setup_interpreter.png)
 
+## Kedro VSCode Extension
```

Comment:
Nit: in most places you refer to `VS Code` with a space but here it's `VSCode`, probably best to pick one for consistency